packages/react-devtools-shared/src/symbolicateSource.js

```diff
@@ -84,29 +85,33 @@ async function symbolicateSource(
             columnNumber, // 1-based
           });
 
+          if (possiblyURL === null) {
+            return null;
+          }
```

Comment:
Any reasons for not throwing here, like it was before?

Reply:
The old logic of `consumer.originalPositionFor` is thrown when either `sourceContent` or `sourceURL` is null, but this requirement is only necessary for parsing hook names.

By removing this restriction, the new logic is to return `sourceContent` and `sourceURL` as-is. I only check if `possiblyURL` is null because we don't care about sourceContent here. The check is moved to the function that parses hook names.

Reply:
And if I throw an error here, it will be caught by the outer try-catch block, which also `return null`. Return null directly is faster.